test/system/src/main/java/io/pravega/test/system/framework/services/PravegaSegmentStoreService.java

```diff
@@ -127,20 +127,22 @@ private App createPravegaSegmentStoreApp() {
         //set env
         String zk = zkUri.getHost() + ":" + ZKSERVICE_ZKPORT;
 
-        //System properties to configure SS service.
-        String hostSystemProperties = setSystemProperty("pravegaservice.zkURL", zk) +
-                setSystemProperty("bookkeeper.zkAddress", zk) +
-                setSystemProperty("hdfs.hdfsUrl", "hdfs.marathon.containerip.dcos.thisdcos.directory:8020") +
-                setSystemProperty("autoScale.muteInSeconds", "120") +
+        //Environment variables to configure SS service.
+        Map<String, String> map = new HashMap<>();
+        map.put("ZK_URL", zk);
```

Comment:
Should `ZK_URL` be in quotes? I thought ZK_URL (and the others below here) were constants.